index.md

```diff
@@ -4,52 +4,48 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
 
-Learn to tame your unruly data processing workflow with
-[Snakemake](https://snakemake.readthedocs.io/en/stable/), a tool for creating
-reproducible and scalable data analyses. Workflows are described via a human
-readable, Python-based language. They can be seamlessly scaled to server,
-cluster, grid, and cloud environments, without the need to modify the
-workflow definition.
+Learn to tame your unruly data processing workflow with [Snakemake][snakemake],
+a tool for creating reproducible and scalable data analyses. Workflows are
+described via a human readable, Python-based language. They can be seamlessly
+scaled to server, cluster, grid, and cloud environments, without the need to
+modify the workflow definition. In this lesson, you will build up a
+reproducible, automated, and efficient workflow step by step with
+Snakemake. Along the way, you will learn the benefits of modern workflow
+engines and how to apply them to your own work.
 
-Starting with some independent analysis tasks, you will explore the
-limitations of manual processing and shell scripting before building up a
-reproducible, automated, and efficient workflow step by step with Snakemake.
-Along the way, you will learn the benefits of modern workflow engines and how
-to apply them to your own work.
-
-The example workflow performs a frequency analysis of several public domain
-books sourced from [Project Gutenberg](https://www.gutenberg.org/), testing
-how closely each book conforms to [Zipf's Law][ref-zipf]. This example has
-been chosen over a more complex scientific workflow as the goal is to appeal
-to a wide audience and to focus on building the workflow without getting
+The example workflow will launch several cluster jobs with the [Amdahl][amdahl]
+program from the [Introduction to High-Performance Computing][hpc-intro] using
+different numbers of processors, collect the output from each job, and create a
+graph of "speedup" (runtime divided by the runtime with just one processor) as
```

Comment:
```suggestion
graph of "speedup" (reference runtime, usually one processor or node or GPU, divided by the runtime with increased resources) as
```

Reply:
Speed-up (hopefully!) goes up from 1, so you need the larger number on top

Reply:
:zany_face: